seed/static/seed/js/controllers/column_settings_controller.js

```diff
@@ -67,9 +67,30 @@ angular.module('BE.seed.controller.column_settings', [])
         {id: 'geometry', label: $translate.instant('Geometry')}
       ];
 
-      $scope.changeText = function(btnText) {
-        if(btnText === 'Collapse Help'){
-          $scope.btnText = 'Expand Help' ;
+      $scope.comstock_types = [
+        {id: null, label: ''},
+        {id: 'division', label: $translate.instant('comstock.division')},
+        {id: 'hvac_system_type', label: $translate.instant('comstock.hvac_system_type')},
+        {id: 'rentable_area', label: $translate.instant('comstock.rentable_area')},
+        {id: 'number_of_stories', label: $translate.instant('comstock.number_of_stories')},
+        {id: 'year_built', label: $translate.instant('comstock.year_built')},
+        {id: 'weekend_start_time', label: $translate.instant('comstock.weekend_start_time')},
+        {id: 'weekend_duration', label: $translate.instant('comstock.weekend_duration')},
+        {id: 'weekday_start_time', label: $translate.instant('comstock.weekday_start_time')},
+        {id: 'weekday_duration', label: $translate.instant('comstock.weekday_duration')},
+        {id: 'building_shape', label: $translate.instant('comstock.building_shape')},
+        {id: 'built_code', label: $translate.instant('comstock.built_code')},
+        {id: 'rotation', label: $translate.instant('comstock.rotation')},
+        {id: 'aspect_ratio', label: $translate.instant('comstock.aspect_ratio')},
+        {id: 'building_type', label: $translate.instant('comstock.building_type')},
+        {id: 'state', label: $translate.instant('comstock.state')},
+        {id: 'county', label: $translate.instant('comstock.county')},
+        {id: 'climate_zone', label: $translate.instant('comstock.climate_zone')}
+      ];
```

Comment:
I attempted this, no dice since constants can't inject provider dependencies.  I used [this trick](https://stackoverflow.com/a/31434481) to inject $translate, but then it couldn't access the $translateProvider.